cmd/pilot-agent/main.go

```diff
@@ -118,24 +136,18 @@ var (
 )
 
 func init() {
+	proxyCmd.PersistentFlags().StringVar(&config, "config", "",
+		"Configuration overrides for the proxy as YAML/JSON")
```

Comment:
Rename this to configValues and add a config-file option. So that we can do the fugly thing behind the scenes but have a nicer option for humans